### PR TITLE
fixed bug on query by removing space between filter values

### PR DESCRIPTION
**Pull request by iamebuka**
---
name: Removing additional spaces in query filter value
about: This pull request handles removing spaces in filters when generating filters for your search
---

**Is your pull request related to an issue? Please describe.**
This pull request is related to issue  #292. Additional spaces in queries throw an error on the server, these spaces could result when entering filter values for your search filters, a user might enter values containing spaces for example " > 1" instead of ">1". 

**List of functionalities contained in this pull request**
A function that parses user filter in the right form and removing spaces in between values

### CODE REVIEW

File: src/app/search/search.component.ts [hunk 1]
```diff
@@ -267,15 +267,29 @@ export class SearchComponent implements OnInit {
 
   applyFilter(row: any) {
     this.model.query += this.parseFilter(row);
+    console.log(this.parseFilter(row))
```
Comment (nguyenLisa) [inline src/app/search/search.component.ts]: Are we keeping this?

Comment (iamebuka) [reply src/app/search/search.component.ts]: @nguyenLisa Yes. that's part of the function that formats the filter so it is acceptable by the server

Comment (nguyenLisa) [reply src/app/search/search.component.ts]: @iamebuka , I meant: is it useful to log this information to the console? If not, then there's no need for this.

Comment (iamebuka) [reply src/app/search/search.component.ts]: Oh yea it isn't necessary. I am removing it

Comment (nguyenLisa) [reply src/app/search/search.component.ts]: Great, thanks! With additional test cases, this should be good to go.

Comment (iamebuka) [reply src/app/search/search.component.ts]: @nguyenLisa I noticed some git error in my commit, I would like to make changes before you continue

Comment (iamebuka) [reply src/app/search/search.component.ts]: Please disregard this pull request, i realized i committed changes i was working on a different branch to this issue branch before pushing to git. Apart from that everything works fine. So i would be closing this and deleting the branch while i issue a new pull request just for this issue

**Comment by ankur-1989** [comment]
@iamebuka  
Do we need this now as PR296 has been merged?

**Comment by iamebuka** [comment]
@ankur-1989 Nope i am closing it now as PR 296 has been merged.

**Pull request closed without merging**